word_guess.rb

```diff
@@ -0,0 +1,192 @@
+require 'colorize'
+
+class WordGuess
+  def initialize
+    puts "FANTASY WORD GUESS GAME"
+    puts "-" * 40
+    @easy_word_array = %w[ tree legend dragon knight night wizard  goblin queen king princess elf dwarf fairy slay honor joust kingdom sword spear forest]
+    @hard_word_array = %w[kingdom centaur sorcerer courageous whimsical petrify leprechauns conjure enchanting cauldron]
+    @already_guessed_letters = []
+    @tally = 0
+    difficulty_level
+    @current_word = @word_array.sample
+    @correct_letters = Array.new(@current_word.length, "_" )
+    ascii
+    check_letter
+  end
+
+  # Do we want to show the user how many tries they have left each turn?
+
+  # fix method call stuff at the bottom
+
+  # handle invalid input for the easy/hard method
+
+  # We should turn this into a loop so that if they accidentally type something
+  # other than easy or hard, it won't break the game
+  def difficulty_level
+
+    puts "What difficulty would you like?"
+    level = nil
+    until level == "hard" || level == "easy"
+      puts "Please choose easy or hard."
+      level = gets.chomp.downcase
+      if level == "hard"
```

Comment:
I think it might make sense to have this conditional _outside_ of the `until` loop because it should not execute until we have a good value for the `level` variable